src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -5,7 +5,9 @@ SVC_NAME=${SVC_NAME// /_}
 SVC_DESCRIPTION="{{SvcDescription}}"
 
 SVC_CMD=$1
-arg_2=${2}
+SVC_RUN_AS_USER=
+
+SCRIPT_NAME=$0
```

Comment:
Can we just read from an environment variable and overwrite `TEMPLATE_PATH` based on that env?
Then we don't need to change the rest of the svc.sh template file, also, we probably want to make same change to `darwin.svc.sh.template` as well.

Reply:
Sure, but then we would have to rely on users remembering that they set the path. But regardless, by using flags, it is more explicit that you want to provide your own template, and it is nice for documentation purposes.

Also, I guess it is more convenient to provide in the command you are starting a service your template than exporting or setting env var and then running it. Let's say I exported a variable to run my template, and I made a mistake. Now, to run the default svc config, I must unset the TEMPLATE_PATH env variable since it would be used again on the next run. With this, I would just run the same command without the flag.

Reply:
`svc.sh` is used when you install/uninstall/start/top the service, but the `TEMPLATE_PATH` is only used during `install`.

I don't think we want the customer to install using 1 template and start the service as a different template.

Also, I am trying to minimize the change as much as possible.

Reply:
Sure, we can do it that way then. I will revert to the previous state and add fix it that way 😊